target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -36,6 +36,7 @@ ramips_setup_interfaces()
 		;;
 	ampedwireless,ally-r1900k|\
 	asus,rt-ax53u|\
+	asus,rt-ax1800hp|\
```

Comment:
This device looks like having 4 lan ports + wan, probably not the best place to define it here, or at all? (since the default is 4 lan ports + wan) ?

Reply:
I will fix later in today